ldap/servers/slapd/back-ldbm/db-mdb/mdb_debug.c

```diff
@@ -434,10 +476,10 @@ int
 dbg_mdb_drop(const char *file, int lineno, const char *funcname, MDB_txn *txn, MDB_dbi dbi, int del)
 {
     int rc = mdb_drop(txn, dbi, del);
-    if (dbgmdb_level & DBGMDB_LEVEL_MDBAPI) {
+    if (dbg_should_log(DBGMDB_LEVEL_MDBAPI, dbi, NULL)) {
         char dbistr[DBGVAL2STRMAXSIZE];
         dbi_str(NULL, dbi, dbistr);
-        dbg_log(file, lineno, funcname, DBGMDB_LEVEL_MDBAPI, "mdb_drop(txn: %p, %s, del: %d)=%d", txn, dbistr, del, rc);
+        dbg_log(file, lineno, funcname, DBGMDB_LEVEL_MDBAPI+DBGMDB_LEVEL_FORCE, "mdb_drop(txn: %p, %s, del: %d)=%d", txn, dbistr, del, rc);
```

Comment:
I think normal convention is | not + for log mask combinations, but it's not a huge deal here I guess.